client/sources/ok_test/models.py

```diff
@@ -100,9 +100,11 @@ def unlock(self, interact):
                                 for case in suite.cases])
         for suite_num, suite in enumerate(self.suites):
             for case_num, case in enumerate(suite.cases):
+                case_id = '{} > Suite {} > Case {}'.format(
```

Comment:
Basically, I was thinking of this as an acceptable use case: Student wants to add another test, so they add a different suite to the beginning of the ok test. I'm assuming that they're leaving our test cases alone, but the "Suite number" could get corrupted in that case.

If a student *really* wanted to mess with this, they could do a ton of stuff- I just want to make sure that good behaviors aren't going to be problems for us.